info.limpet.stackedcharts.ui/src/info/limpet/stackedcharts/ui/editor/figures/ChartsetFigure.java

```diff
@@ -26,9 +26,9 @@ public ChartsetFigure(ActionListener addChartHandler)
     chartsetHeader.setTextAlignment(PositionConstants.TOP);
     if (boldFont == null)
     {
```

Comment:
@dinkoivanov - I had to change the display from Current to Default for the next two lines, to avoid an NPE error when restoring a session with an empty Stacked Charts View visible.  I can't claim to completely understand why one works and the other doesn't. But, is it ok with you?